### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by omahs**
Fix typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ For more detailed information about how to connect and use the testnet network,
 - 5GB of free disk space, an SSD is highly recommended (100GB or more is required for history nodes)
 - File systems with sparse file support.
 - It is highly recommended to run a clock synchronization protocol such as NTP. This
-  is needed for properly accept blocks according to the timestamp and it is specially
+  is needed for properly accept blocks according to the timestamp and it is especially
```
Comment (sisou) [inline README.md]:
```suggestion
  is needed to properly accept blocks according to the timestamp and it is especially
```

File: primitives/account/src/account/staking_contract/staker.rs [hunk 2]
```diff
@@ -384,7 +384,7 @@ impl StakingContract {
                 return Err(AccountError::InvalidForRecipient);
             }
 
-            // Fail if the the funds are in locked or jailed.
+            // Fail if the funds are in locked or jailed.
```
Comment (sisou) [inline primitives/account/src/account/staking_contract/staker.rs]:
```suggestion
            // Fail if the funds are locked or jailed.
```

**Review by sisou — COMMENTED** [review]
Much effort, such wow.

Please squash all commits into one.

**Comment by omahs** [comment]
@sisou Thanks for the quick review! Very squashed

**Review by jsdanielh — APPROVED** [review]

**Comment by jsdanielh** [comment]
Signed commit to merge it

**Pull request merged**